### DISCUSSION TITLE
Is `evaluation.py` fixed and cannot be changed?

### DISCUSSION THREAD

**Discussion by bluetex315** [PaddlePaddle/PaddleSpatial#215] [BaiduKDDCup2022] [answered]
Dear Committee:

I am wondering if we can change the `evaluation.py` file? Or it is fixed at the backend. Also, could you please hint on whether we can upload the prediction of all wind turbines as a whole rather than predicting one by one as in the evaluation sample provided. Many thanks.

**Comment by ramber1836** [comment, marked as answer]
The evaluation is fixed at the backend. 
You can predict the wind power as a whole and produce the outcome predictions with the shape in accordance with the requirement, i.e. (134,288,1) by averaging the predicted whole power over 134 turbines.
